### DISCUSSION TITLE
Should "App-test.js" be renamed to "App.test.js"?

### DISCUSSION THREAD

**Discussion by Adnan-Bacic** [react-native-community/discussions-and-proposals#570] [Ideas]
when creating a react project with `create-react-app`, the starter test file for the `App.js` file is called `App.test.js`

in react-native, the starter test file for `App.js` is called `App-test.js`

i feel like it would make sense if react and react-native unified the naming scheme for this. opinions?

**Comment by thymikee** [comment]
It's in `__tests__` directory so the name doesn't really matter (at least from Jest perspective). Plus at this point it's a throw-away test with pretty inaccurate title "renders correctly" (instead of "renders whole app without crashing").

Nevertheless, I'd personally prefer the _App.test.tsx_ name, with TS included. And if we ever decide to change the name, I'd take this opportunity to add some meaningful test, e.g. with React Native Testing Library.

**Reply by NickGerleman** [reply]
FYI it did get converted to tsx for 0.71 (though no other changes).

**Comment by NickGerleman** [comment]
Change makes sense to me, as more aligned with Jest conventions (though as pointed out Jest will interpret anything in the tests directory as a test). https://jestjs.io/docs/configuration#testregex-string--arraystring

**Comment by Adnan-Bacic** [comment]
pr merged: https://github.com/facebook/react-native/pull/35826

**Reply by kelset** [reply]
thanks for your help 👏
